prow/cmd/peribolos/main.go

```diff
@@ -255,19 +255,78 @@ func configureOrg(opt options, client *github.Client, orgName string, orgConfig
 
 	// teams = set(all teams)
 	for name, team := range orgConfig.Teams {
-		team.Name = name
-		num := -1
-		// find team name in teams (or previous name)
-		if err := configureTeam(client, num, team); err != nil {
+		_ = name
+		if err := configureTeam(client, orgName, team); err != nil {
 			return fmt.Errorf("failed to update %s: %v", orgName, err)
 		}
 	}
 	return nil
 }
 
-func configureTeam(client *github.Client, num int, team org.Team) error {
-	// create team if num < 0
-	// else diff and patch
+func findTeam(client *github.Client, org, name string, previousNames ...string) (*github.Team, error) {
```

Comment:
We're looping over this so we do the `ListTeams` call a large amount of times. Could build a `map[string]Team` on team name and do one client call instead

Reply:
Done